geode-core/src/test/java/org/apache/geode/management/internal/security/CacheServerMBeanAuthorizationJUnitTest.java

```diff
@@ -95,21 +95,21 @@ public void testNoAccess() throws Exception {
     SoftAssertions softly = new SoftAssertions();
 
     softly.assertThatThrownBy(() -> bean.removeIndex("foo"))
-        .hasMessageContaining(TestCommand.dataManage.toString());
+        .hasMessageContaining(ResourcePermissions.DATA_MANAGE.toString());
     softly.assertThatThrownBy(() -> bean.executeContinuousQuery("bar"))
-        .hasMessageContaining(TestCommand.dataRead.toString());
+        .hasMessageContaining(ResourcePermissions.DATA_READ.toString());
     softly.assertThatThrownBy(() -> bean.fetchLoadProbe())
-        .hasMessageContaining(TestCommand.clusterRead.toString());
+        .hasMessageContaining(ResourcePermissions.CLUSTER_READ.toString());
     softly.assertThatThrownBy(() -> bean.getActiveCQCount())
-        .hasMessageContaining(TestCommand.clusterRead.toString());
+        .hasMessageContaining(ResourcePermissions.CLUSTER_READ.toString());
     softly.assertThatThrownBy(() -> bean.stopContinuousQuery("bar"))
         .hasMessageContaining(TestCommand.clusterManageQuery.toString());
```

Comment:
Should `CLUSTER_MANAGE_QUERY` et al be added to `ResourcePermissions` and referenced as a common permission?  Or is just having a two-level resolution pre-defined sufficient for must use cases, allowing the three-level case to be explicitly defined at the point of use?

Reply:
Let's tackle this question when we do the work to re-introduce String methods into SecurityService